src/hooks/useApiAdminCircle.ts

```diff
@@ -17,8 +17,8 @@ export const useApiAdminCircle = (circleId: number) => {
   const { fetchManifest } = useApiBase();
 
   const updateCircle = useRecoilLoadCatch(
-    () => async (params: PutCirclesParam) => {
-      await getApiService().putCircles(circleId, params);
+    () => async (params: ValueTypes['circles_set_input']) => {
+      await mutations.updateCircle(circleId, params);
```

Comment:
`{"errors":[{"extensions":{"details":[{"path":["input","payload","discord_webhook"],"code":"invalid_string","message":"Invalid url","validation":"url"}],"code":"422"},"message":"Invalid input: Invalid url (discord_webhook)"}]}`

Getting this error when i click on save, i think the issue is the `discord webhook` value is empty before `edit webhook` button is clicked

Reply:
thanks @zashton this was a good catch. I've changed it to handle it in the way that laravel did, with an `update_webhook` flag. Also handling the empty string as null so we can un-set it.